### PR TITLE
Copy files for my request review project

### PR DESCRIPTION
**Pull request by yash244466666**
Copied files from fellow Micronauts project because my projects are not available to me now.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#image {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    margin: 0 auto;
```
Comment (yash244466666) [inline css/style.css:5]: Please add only two spaces before writing the CSS properties. That way, your code will look more professional. 😎😎😎

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      <button>
+        <a
+          id="tribute-link"
+          href="https://github.com/PeteUgwu/nabel-and-peter-.git"
+          target="_blank"
+          >see more info</a
+        >
+      </button>
+    </main>
+    <script src="js/index.js"></script>
```
Comment (yash244466666) [inline index.html:68]: Please use "Class Selector" more often.  You have used  "ID selector" everywhere. IDs should be used for unique items on the page. 😊😊😊😎

**Review by yash244466666 — COMMENTED** [review]
### **STATUS: REQUIRED CHANGES ♻️♻️**
Hi @yash244466666
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there!

143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91


To highlight 🍾
All linters are passing ✔️
Good commit history ✔️
Your design looks good 💯 ✔️
Good PR description Follow GitHub flow ✔️
Required changes ♻️ ♻️
 Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @yash244466666  when you comment so I can receive the notification or use slack with the same name.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mckent05 — APPROVED** [review]
Hi @yash244466666 , 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and welldone! 🎉 

## Highlights
- Good job being polite while giving your reviews 👏 
- You have done well stating the reasons for suggesting a change 👍 
- 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**